src/main/java/com/cazsius/solcarrot/SOLCarrotConfig.java

```diff
@@ -129,20 +135,20 @@ public static class Server {
 			milestones = builder
 				.translation(localizationPath("milestones"))
 				.comment("A list of numbers of unique foods you need to eat to unlock each milestone, in ascending order. Naturally, adding more milestones lets you earn more hearts.")
-				.defineList("milestones", Lists.newArrayList(5, 10, 15, 20, 25), e -> e instanceof Integer);
+				.defineList("milestones", Lists.newArrayList(5, 10, 15, 20, 25), () -> 5, e -> e instanceof Integer);
```

Comment:
What's this 5 about? A default value for newly added entries?

Reply:
Yeah, when using the built in Neoforge config screen

Reply:
Hm, I'm a bit worried that this would increase misunderstandings of the mechanism: if people added 5 to the end of the default list, it would be invalid—in order to add a new milestone 5 foods higher than the last one, they'd have to add 30. There's not really a way to encode this I fear, so maybe something like 0 would be more appropriate?

We could also think about changing the format entirely and instead asking for the amounts between each milestone, making the default list `[5, 5, 5, 5, 5]` and 5 a perfectly reasonable default addition. This NeoForge port might be a good time to change that!

Reply:
I'll change it to 0 for now, then you can change it later